### ISSUE TITLE
Unstable to use in Visual Studio 2017 RC + core 1.1

### ISSUE THREAD

**Issue by Biarity** [aspnet/Tooling#911]
1. Install VS 2017 RC 
2. Install .NET Core 1.1
3. Create a new ASP.NET Core project
4. Select the target framework to be netcoreapp1.1
5. Update all the nuget packages that come with it to 1.1.0
6. Get an error while updating 🙁

**Comment by mlorbetske** [comment]
Hi @Biarity could you add the error message to this issue please?

**Comment by Biarity** [comment]
Note that I have .NET Core 1.1 (current release) installed from dot.net. I get this window:
![image](https://cloud.githubusercontent.com/assets/3658686/21409489/533ed80a-c826-11e6-912a-98cf513aecf5.png)

Then in the solution, I get this weird folder:
![image](https://cloud.githubusercontent.com/assets/3658686/21409526/86f912a0-c826-11e6-8d56-695166d5955b.png)

I heard that having VS 2015 installed side-by-side ma cause some errors, so I uninstalled it. I used to get an error that `Microsoft.Extensions.Logging` is out of date but I no longer do now, however, the previously mentioned errors still persist.

Note that if I ignore these errors everything works fine, so I'm not sure why they're there in the first place.
